src/get-severity.ts

```diff
@@ -0,0 +1,28 @@
+import { Severity, TodoData } from './types';
+
+/**
+ * Returns the correct severity level based on the todo data's decay dates.
+ *
+ * @param todo - The todo data.
+ * @param today - A number representing a date (UNIX Epoch - milliseconds)
+ * @returns Severity - the lint severity based on the evaluation of the decay dates.
+ */
+export function getSeverity(todo: TodoData, today: number = getDatePart().getTime()): Severity {
+  if (todo.errorDate && today > todo.errorDate) {
+    return Severity.error;
+  } else if (todo.warnDate && today > todo.warnDate) {
+    return Severity.warn;
+  }
```

Comment:
Should it be `today >= todo.errorDate`? Specifically when I think about the dates, I think if todays date _is_ the "error date" that it should change things into an error. 

Super minor detail here, I think either choice is perfectly valid. Feel free to resolve if you prefer the current logic.

Reply:
I was thinking of it as being passed the date, not on the date. I sorta think that's the correct way to think about this, but am open.

Reply:
kk, lets keep it as is and then address if we have confusion from folks during adoption.